Formula/fileicon.rb

```diff
@@ -0,0 +1,16 @@
+class Fileicon < Formula
+  desc "macOS CLI for managing custom icons for files and folders"
+  homepage "https://github.com/mklement0/fileicon"
+  url "https://github.com/mklement0/fileicon/archive/v0.2.2.tar.gz"
+  sha256 "1725a6a693b3586617b0fc669719a15c05eb7f8e535daee511a315d9b6fc0eb3"
+
+  def install
+    bin.install "bin/fileicon"
```

Comment:
Why not use the makefile in the directory?

Reply:
@SMillerDev I keep getting the following error, preventing me:
~~~ sh
Error: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or
https://developer.apple.com/download/more/.
~~~

Any ideas? 😕

Reply:
Update your command line tools 😄. Updates unfortunately don't always show up in system preferences. You might want to check the URL for a download

Reply:
@SMillerDev Thanks… is there some way I can force the update? 😅

Reply:
@SMillerDev Also, there seems to be no `install:` or `build:` target in the makefile… so what do you mean exactly…?

Reply:
Don't think you can force the update. As for the makefile, I just saw there was one in the repo. I didn't actually check the content. You should make a pull request for an install step in it, but for now this will also work

Reply:
@SMillerDev I just finally got the update from Apple… guess they are a little slow sometimes. So this is good to go for now?

Reply:
Could you still make a pull request upstream to make the makefile more useful?